src/commonTest/kotlin/com/sunnychung/lib/multiplatform/kdatetime/KDateTimeFormatTest.kt

```diff
@@ -79,6 +79,12 @@ class KDateTimeFormatTest {
             assertEquals(0, dateTime.zoneOffset.hours)
             assertEquals(0, dateTime.zoneOffset.minutes)
         }
+
+        KDateTimeFormat("yy-MM-dd hh:mm:ssaaZ").apply { setAmPmNames("오전", "오후") }.parseToKZonedDateTime("23-09-11 02:54:19오후UTC").toKZonedInstant().let { dateTime ->
+            assertEquals(1694444059000, dateTime.toMilliseconds())
+            assertEquals(0, dateTime.zoneOffset.hours)
+            assertEquals(0, dateTime.zoneOffset.minutes)
+        }
```

Comment:
We can add more test cases on parsing am/pm, covering non-caps, caps and mixed cases, just like what you have implemented in the `formatAmPm` test.

As the code of testing am/pm parsing would grow larger, this test can be extracted out as another `@Test` function.

Reply:
then, I'll move these lines to `formatAmPm` method.